evacuees/src/UI/embc-evacuee/src/app/sharedModules/components/confirm-restriction/confirm-restriction.component.scss

```diff
@@ -1,3 +1,13 @@
 .needs-button {
     width: 50%;
-}
\ No newline at end of file
+}
+
+.content-container {
+    padding: 0 24px 24px 24px;
+}
+
+@media (max-width: 768px) {
```

Comment:
this style is duplicate. The same one already must exist in styles.css. Can you please check?